net/arp-scan/Makefile

```diff
@@ -43,4 +43,40 @@ define Package/arp-scan/install
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/arp-scan $(1)/usr/bin/
 endef
 
+define Package/arp-scan/postinst
+cat <<EOF
+
+Please install the arp-scan-database package in order to let arp-scan
+display additional information about vendor/manufacturer for each
+discovered device.
+
+EOF
+endef
+
 $(eval $(call BuildPackage,arp-scan))
+
+define Package/arp-scan-database
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=MAC database for ARP scanner
+  DEPENDS:=+arp-scan
+  URL:=https://github.com/royhills/arp-scan
+  PKG_MAINTAINER:=Sergey Urushkin <urusha.v1.0@gmail.com>
```

Comment:
We might add `define Package/arp-scan/default` and use `$(Package/arp-scan/default)` in a particular package. 
Also, `PKG_MAINTAINER` seems to be redundant here.
And arp-scan-database actually doesn't depend on arp-scan, it doesn't even require fixup/configuring/building, just copying, but I'm not sure if we should skip this for a single package (and have such ability). @feckert please advice, what is a better way to accomplish this?

Reply:
- This would be good to move this to `define Package/arp-scan/default`
- Also`PKG_MAINTAINER` should be remove from the `Package/arp-scan/` definition
- I think the dependency is not necessary.